main.js

```diff
@@ -198,4 +204,37 @@ const formCheck = () => {
   });
 };
 
-formCheck();
\ No newline at end of file
+formCheck();
+
+// preserve data in the browser
+body.onload = () => {
+  if (recievingData) {
+    const dataSave = JSON.parse(recievingData);
+    nameValue.value = dataSave.name;
+    emailInput.value = dataSave.email;
+    msg.value = dataSave.message;
+  }
+};
+
+document.querySelectorAll('input').forEach((input) => {
```

Comment:
- You did a great job by using  a queryselectorAll to select  your  all  input but for the message, you didn't use the input you use the textarea , your name and email are saved correctly in the local storage as required but the massage  is not saving as it showing bellow.   :-1: :smiley:  :smiley: 

![image](https://user-images.githubusercontent.com/95297251/190860217-7c1cbdac-f4c0-4388-b12e-9c665bf7f761.png)
  
- [ ] Kindly make sure you fix this error, the simple to fix this error is to add textarea in your queryselectorAll().check below the format. ::smiley:  :smiley: :100: :+1:  
`document.querySelectorAll('input, textarea').forEach((input) => {`

Reply:
@MaganaAsiati Thanks a lot 🙌 for your efforts in reviewing my code, I really appreciate it😊.